packages/core/src/tag/tag.stories.ts

```diff
@@ -82,7 +81,6 @@ export function badgesStatus() {
       <cds-tag readonly status="danger"
         >Danger <cds-badge aria-label="notification 12" status="danger">12</cds-badge></cds-tag
       >
-      <cds-tag disabled status="info">Disabled <cds-badge aria-label="notification 12">12</cds-badge></cds-tag>
```

Comment:
Thanks for removing, the disabled was a bit confusing in the readonly status colors. 👍